### PR TITLE
Pattern-1 Issues

### PR DESCRIPTION
**Pull request by BLasan**
**Changes**

- Added common configs for analytics
- Added missing volumes & volume mounts in analytics
- Removed pvc
- Introduces synchronization for artifacts

### CODE REVIEW

File: artifacts/api-manager-artifacts/pattern-1/wso2am-p1-apim-1-conf.yaml [hunk 1]
```diff
@@ -219,8 +219,8 @@ data:
     #port = 10005
 
     [[apim.throttling.url_group]]
-    traffic_manager_urls = ["tcp://localhost:9611"]
-    traffic_manager_auth_urls = ["ssl://localhost:9711"]
+    traffic_manager_urls = ["tcp://wso2-am-1-svc:9611"]
+    traffic_manager_auth_urls = ["ssl://wso2-am-1:9711"]
```
Comment (pubudu538) [inline artifacts/api-manager-artifacts/pattern-1/wso2am-p1-apim-1-conf.yaml]: This has to be wso2-am-1-svc:9711. I think we need to revisit service names that has used in here.

File: pkg/controller/controller.go [hunk 2]
```diff
@@ -400,23 +406,28 @@ func (c *Controller) syncHandler(key string) error {
 			}
 		}
 
+		klog.Info("AM-2 Config")
```
Comment (pubudu538) [inline pkg/controller/controller.go]: These log lines do not provide any useful information. We have to improve these log lines

**Review by pubudu538 — CHANGES_REQUESTED** [review]

File: pkg/controller/controller.go (hunk 2 above)
Comment (BLasan) [reply pkg/controller/controller.go]: I'll remove them.

**Comment by BLasan** [comment]
@pubudu538 Changes added. Please review

File: artifacts/api-manager-artifacts/pattern-1/wso2am-p1-analytics-dash-bin.yaml [hunk 3]
```diff
@@ -17,7 +17,7 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: wso2am-p1-analytics-dash-bin
+  name: wso2am-p1-analytics-bin
```
Comment (pubudu538) [inline artifacts/api-manager-artifacts/pattern-1/wso2am-p1-analytics-dash-bin.yaml]: Remove p1 in here.

File: artifacts/api-manager-artifacts/pattern-1/wso2am-p1-analytics-dash-bin.yaml [hunk 4]
```diff
@@ -306,7 +306,7 @@ data:
     do
         $JAVACMD \
         -Xbootclasspath/a:"$CARBON_XBOOTCLASSPATH" \
-        -Xms256m -Xmx1024m \
+        -Xms2048m -Xmx2048m \
```
Comment (pubudu538) [inline artifacts/api-manager-artifacts/pattern-1/wso2am-p1-analytics-dash-bin.yaml]: Change to default values.

**Review by pubudu538 — APPROVED** [review]

**Pull request merged**